src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -237,6 +237,11 @@ and whether it is an `isPrimary` one. Note that the built-in screen
 The `currentScreen` field is a live object corresponding to the current `window.screen`. The object
 is updated on cross-screen window placements or device changes.
 
+{% Aside %}
+Earlier versions of the API would not return detailed screen labels like `"Built-in Retina
```

Comment:
Note that it can still return less descriptive names if the platform does not provide friendly names. 
See https://chromium-review.googlesource.com/c/chromium/src/+/3651733/2/third_party/blink/renderer/modules/screen_enumeration/screen_detailed.cc#b102

Reply:
Actually we've since updated the code to always return the platform name, even if it's empty... As sites may wish to construct their own placeholder names for empty labels, as our partner requested. https://crsrc.org/c/third_party/blink/renderer/modules/screen_enumeration/screen_detailed.cc;drc=1e615e67d32d66e20176d8be220c11aef6b99cf6;l=144

Reply:
Could I ask you to propose a way to formulate this? Currently I have `Earlier versions of the API would not return detailed screen labels like "Built-in Retina Display" given the underlying platform provides them, but only less descriptive ones like "Internal Display 1"`.

Reply:
Thanks for adopting and fixing up my suggestion in the other comment thread.